internal/api/core/describe.go

```diff
@@ -449,6 +507,19 @@ func describeStruct(customType reflect.Type, knownPackages map[string]StructDesc
 	return description, nil
 }
 
+func isFieldRequired(field reflect.StructField) bool {
+	tag, ok := field.Tag.Lookup("required")
+	if !ok {
+		return false
+	}
+
+	if tag == "" {
+		return true
+	}
+
+	return false
```

Comment:
This silent failure seems a bit dangerous.
Unless I am missing something else,
a user can put in a bad tag (e.g.: `required:"true"`) and not have the field be required (unbeknownst to them).